deno.json

```diff
@@ -1,9 +1,17 @@
 {
+  "name": "@cmd-johnson/deno-oauth2-client",
+  "version": "1.0.3",
   "imports": {
+    "@oak/oak": "jsr:@oak/oak@^17.1.2",
+    "@std/assert": "jsr:@std/assert@^1.0.6",
+    "@std/encoding": "jsr:@std/encoding@^1.0.5",
+    "@std/http": "jsr:@std/http@^1.0.9",
+    "@std/testing": "jsr:@std/testing@^1.0.3",
     "https://deno.land/x/oauth2_client/": "./"
   },
   "tasks": {
     "check:types": "deno check **/*.ts",
     "test": "deno test --parallel --trace-leaks --doc"
-  }
+  },
+  "exports": "./mod.ts"
```

Comment:
This means `@cmd-johnson/deno-oauth2-client` is the only entry point. Alternatively, we could also include more specific modules. E.g.:

- `@cmd-johnson/deno-oauth2-client/errors`
- `@cmd-johnson/deno-oauth2-client/pkce`
- `@cmd-johnson/deno-oauth2-client/refresh-token-grant`